Lib/lua/std_except.i

```diff
@@ -19,9 +19,9 @@ namespace std
   class exception 
   {
   public:
-    exception() throw() { }
-    virtual ~exception() throw();
-    virtual const char* what() const throw();
+    exception() noexcept { }
+    virtual ~exception() noexcept;
+    virtual const char* what() const noexcept;
   }; 
 }
```

Comment:
This is only shown to SWIG and the `%ignore` should mean no code gets generated.  However we can rely on SWIG supporting `noexcept` so this change should be safe and perhaps worthwhile even if it doesn't actually do anything for the stated goal here.